### DISCUSSION TITLE
v2.14.0

### DISCUSSION THREAD

**Discussion by sojan-official** [chatwoot/chatwoot#6468] [Releases & Announcements]
## Changelog

- Conversation Participants
- Support for authentication via Google Oauth
- Google Translate Integration
- Image support in help centre articles
- UI for selecting Template variables
- Support contact attachments in Whatsapp Channel
- Support sending multiple attachments in Whatsapp Channel
- Ability to impersonate a user from Super Admin
- Ability to update Whatsapp Channel API key from UI
- Instance health status in Super Admin
- New Languages : Icelandic(is) 
- Numerous bug fixes and enhancements

###
Thanks to @iprithvitharun, @fgrep, @opencartcomvn, @imskr,  @diivi, @brunosfg, @jacsonsantospht, @JanWarlen, @gitstart, @BikashSah999, @raph941, @phunguyenmurcul   for the contributions

<hr /><em>This discussion was created from the release <a href='https://github.com/chatwoot/chatwoot/releases/tag/v2.14.0'>v2.14.0</a>.</em>

**Comment by sojan-official** [comment]
## 2.14.0 Dev Changelog

### API Endpoints

#### Additions
- `api/v1/accounts/{account.id}/conversations/<id>/participants
     - https://github.com/chatwoot/chatwoot/pull/6462
     - Ability to add participants to the conversation who will get notifications for new messages.

#### Deletions
- none

---

### Other Changes

- none

**Comment by jochenstu** [comment]
Very nice, thanks!

FWIW on 4GB of RAM (Ubuntu) I ran out of memory and into segmentation faults twice while trying to upgrade (even with added swap). If you are working with 4GB + swap, try setting the [swappinness](https://linuxhint.com/customize-linux-swappiness/) to maybe 80 before running the upgrade.

**Comment by duongvanphung** [comment]
I see a UI bug in message menu context

![image](https://user-images.githubusercontent.com/49987543/219581655-d344f6d4-bd06-4cb8-986a-aeee089d658f.png)

**Comment by netizen-id** [comment]
Please add Live Chat widget with help center. Chenge the UI in Live chat widget 😀

![IMG_20230218_001334](https://user-images.githubusercontent.com/83263168/219707117-b91fa66e-555c-4cff-86bd-f47c309a49b4.jpg)

**Comment by Leandroledledled** [comment]
Hello, I'm using Chattwoot on Cloudron.

Will this update be released to Cloudron automatically? Or do I need to install manual via terminal?

When I click to update it says it's already on the latest version.
![Screenshot_16](https://user-images.githubusercontent.com/126264017/221205514-93777e29-158b-414f-86cf-29ce5e13e6e5.jpg)
